.github/workflows/test_generator_backend.yml

```
@@ -0,0 +1,123 @@
+        id: find_related_test_files
+        run: |
+          cp tools/test_generator/backend/scripts/find_related_test_files.py ./find_related_test_files.py
+          
+          chmod +x find_related_test_files.py
+          
+          python find_related_test_files.py backend_relevant_files.txt $GITHUB_WORKSPACE
+          
+          if [ -s all_test_files.txt ]; then
+            echo "===== List of all test files to be analyzed ====="
+            cat all_test_files.txt
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)